### PR TITLE
Add support to deploy performance dashboards

### PR DESCRIPTION
**Pull request by chaitanyaenr**
### Description
This commit enables users to deploy grafana with preloaded dashboards
visualizing metrics from in-cluster prometheus.

### CODE REVIEW

**Comment by paigerube14** [comment]
rerun

File: env.sh [hunk 1]
```diff
@@ -11,3 +11,4 @@ export PUBLISH_KRAKEN_STATUS=${PUBLISH_KRAKEN_STATUS:=False}
 export PORT=${PORT:=8081}
 export LITMUS_VERSION=${LITMUS_VERSION:=v1.13.8}
 export SIGNAL_STATE=${SIGNAL_STATE:=RUN}
+export DEPLOY_DASHBOARDS=${DEPLOY_DASHBOARDS:=True}
```
Comment (paigerube14) [inline env.sh:14]: in all the docs you put deploy_dashboards default to false and then set it to true here.. think it should be false as default

Comment (chaitanyaenr) [reply env.sh]: Fixed it, thanks.

**Comment by paigerube14** [comment]
Thoughts on going ahead with this pr and fixing the litmus scenarios in PR: https://github.com/cloud-bulldozer/kraken-hub/pull/31

**Comment by chaitanyaenr** [comment]
rerun

**Review by paigerube14 — APPROVED** [review]
LGTM

**Comment by chaitanyaenr** [comment]
rerun

**Comment by chaitanyaenr** [comment]
rerun

**Comment by chaitanyaenr** [comment]
rerun

**Comment by paigerube14** [comment]
rerun

**Comment by paigerube14** [comment]
One failure is due to this known issue: https://github.com/cloud-bulldozer/kraken/issues/176 

i'm good with adding this till the bug is fixed

**Pull request merged**